sdk/storage/storage-blob/test/blobbatch.spec.ts

```diff
@@ -673,3 +674,102 @@ describe("BlobBatch", () => {
     assert.ok(exceptionCaught);
   });
 });
+
+// Won't record this test suite since UUID is randomly generated within the SDK and used in the HTTP request and cannot be preserved.
```

Comment:
Not recording, in this case, is fine, but do call the `skip()` from the test to skip the test in record and playback modes.
Or else the build would fail since we don't have the credentials to run the tests with in the CI in playback mode.

Reply:
Is there a way to skip a test suite as a whole in record and playback modes?
I did this because it's useless to record the `beforeEach` and `afterEach` since all cases in the test suite will be skipped.

Reply:
Is there a way to skip a test suite as a whole? It's useless to run the `beforeEach` and `afterEach` since all cases are skipped.

Reply:
That's a good idea, logged an issue for now https://github.com/Azure/azure-sdk-for-js/issues/12740